.github/workflows/channels-ci.yml

```diff
@@ -286,7 +276,7 @@ jobs:
         wget https://raw.githubusercontent.com/mattermost/mattermost-server/master/build/entrypoint.sh
         chmod +x entrypoint.sh
         export MM_PACKAGE=https://pr-builds.mattermost.com/$REPO_NAME/commit/${{ github.event.pull_request.head.sha || github.sha }}/mattermost-enterprise-linux-amd64.tar.gz
-        docker buildx build --push --build-arg MM_PACKAGE=$MM_PACKAGE -t mattermost/mm-ee-test:${TAG} --provenance=false . 
-  run-performance-bechmarks:
-    uses: ./.github/workflows/performance-benchmarks.yml
-    needs: prepare-docker-build
+        docker buildx build --push --build-arg MM_PACKAGE=$MM_PACKAGE -t mattermost/mm-ee-test:${TAG} --provenance=false .
+  # run-performance-bechmarks:
```

Comment:
There'll be a followup PR to re-enable these when I get the server actions packaging the web app correctly